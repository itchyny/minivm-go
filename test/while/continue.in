a = 1
while a < 10
  print a
  a = a + 1
  if a > 5
    continue
  end
  print 2 * (a - 1)
end
print 100
