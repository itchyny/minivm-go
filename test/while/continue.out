1
2
2
4
3
6
4
8
5
6
7
8
9
100
